backend/app/usecase/requester/verifier.go

```diff
@@ -1,22 +1,27 @@
 package requester
 
-// Verifier verifies in coming network to prevents cyber attacks.
-type Verifier struct {
+// Verifier verifies incoming network to prevent cyber attacks.
+type Verifier interface {
+	IsHuman(recaptchaResponse string) (bool, error)
+}
+
+// VerifierReCaptcha verifies incoming network using ReCaptcha to prevent cyber attacks.
```

Comment:
I capitalized reCaptcha in the doc comments, but should it be lowercase? Or does it not matter?

Reply:
I see the r capitalized in other comments so gonna leave it for now.